### PR TITLE
React Capstone Project

### PR DESCRIPTION
**Pull request by rominana**
In this project I applied the following requirements:
- Use React documentation.
- Use React components.
- Use React props.
- Use React Router.
- Connect React and Redux.
- Handle events in a React app.
- Write integration tests with a React testing library.
- Use styles in a React app.
- Use React life cycle methods.
- Apply React best practices and language style guides in code.
- Use store, actions, and reducers in React.

Live demo : https://react-casptone-rominana.netlify.app/
Video demo: https://www.loom.com/share/799bc65514574cc9bb2852eff8c31acf

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { Route, Routes } from 'react-router-dom';
```
Comment (mateo951) [inline src/App.js:1]: You successfully created a single page application (SPA) using react-router-dom to serve each page without reloading 😎

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import Navbar from './components/Navbar';
+
+function App() {
+  return (
+    <>
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/countries/:regionCode" element={<Countries />} />
+        <Route path="/details/:countryCode" element={<Details />} />
```
Comment (mateo951) [inline src/App.js:15]: The category item detail page has its unique route within the SPA 👍

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi @rominana, 

📝👌 **Your project highlights** 😎 
- General Requirements 💪🤓
- Great Video Presentation ✔
- App deployed using Netlify 🚀
- Implementation of API 🤯

Excelente work! I had a great time reviewing your project 😎 Really liked the GPA API you chose and the final product 🙌 There is a couple of issues that you still need to work on to go to the next project but you are almost there!

Good job so far!
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by mateo951** [comment]
Don't forget that the capstone goal is to build a **mobile** web application to check a list of metrics 🔥 Your app is working as expected technically speaking but is lacking the mobile design. As you can see, there are a couple of issues to work on so that it becomes responsive. 

Your design 
image

Design Guidelines 
image

Please be sure that your design resembles as much as possible the one in the requirements 🤓

**Comment by rominana** [comment]
All the required changes have been applied :smile: 

For this project, I have based on the responsive size that comes by default in the DevTools:
Screenshot from 2022-08-31 22-39-09

Full Mobile Screen screenshot:
react-capstone-mobile

Mobile menu:
menu

**Review by rotimiazeez — APPROVED** [review]
# Hi @rominana 👍🏽 , 
### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**